app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -1288,6 +1288,7 @@ public void onSecurityChanged(@NonNull Session session, boolean isSecure) {
 
         @Override
         public void onUrlChanged(@NonNull Session session, @Nullable String url) {
+            chromeViewModel.onFocusedSessionChanged(url);
```

Comment:
calling onFocusedSEssionChanged() in onUrlChanged() seems weird to me. Maybe there's a better naming for onFocusedSessionChanged

Reply:
updated to onFocusedUrlChanged